elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/ColumnContext.java

```diff
@@ -97,14 +98,12 @@ public Object get(Object key) {
             return value;
         }
 
-        ColumnProjection column = this.getQueryable().getColumnProjection(keyStr);
+        ColumnProjection column = this.getQueryable().getSource().getColumnProjection(keyStr);
         if (column != null) {
 
             ColumnProjection newColumn = column.withArguments(
-                            getColumnArgMap(this.getQueryable(),
-                                            column.getName(),
-                                            this.getColumn().getArguments(),
-                                            emptyMap()));
+                            mergedArgumentMap(column.getArguments(),
+                                              this.getColumn().getArguments()));
```

Comment:
Getting column projection from queryable's source itself, this column projection will have default/available arguments for the column.  So no need to go-to source for default/available arguments again.